core/src/main/java/nl/nn/adapterframework/util/Misc.java

```diff
@@ -1354,5 +1355,25 @@ public static String urlDecode(String input) {
 			return null;
 		}
 	}
-	
+
+	public static <T> void addToSortedListUnique(List<T> list, T item) {
+		int index = Collections.binarySearch(list, item, null);
+	    if (index < 0) {
+	        list.add(Misc.binarySearchResultToInsertionPoint(index), item);
+	    }
+	}
+
+	public static <T> void addToSortedListNonUnique(List<T> list, T item) {
+		int index = Misc.binarySearchResultToInsertionPoint(Collections.binarySearch(list, item, null));
+	    list.add(index, item);		
+	}
+
+	private static int binarySearchResultToInsertionPoint(int index) {
+		// See https://stackoverflow.com/questions/16764007/insert-into-an-already-sorted-list/16764413
+		// for more information.
+	    if (index < 0) {
```

Comment:
Dit bedoelde ik inderdaad. 
Zou je kunnen zorgen dat de indentatie klopt? Er zijn nu een aantal regels die te veel ingesprongen zijn.

Reply:
Ik denk dat er een aantal regels te weinig zijn ingesprongen. Dit komt omdat (o.a.) GitHub vind dat een `tab` niet gelijk is aan 4 spaties. De spaties moeten dus ook tabs worden zodat het overal gelijk is

Reply:
Gefixed.